### PR TITLE
Prepare for merge to master with latest corebundle

### PR DESCRIPTION
**Pull request by rjzondervan**

### CODE REVIEW

File: api/config/bundles.php [hunk 1]
```diff
@@ -20,6 +20,5 @@
     CommonGateway\CoreBundle\CoreBundle::class                                 => ['all' => true],
     Endroid\QrCodeBundle\EndroidQrCodeBundle::class                            => ['all' => true],
     Symfony\Bundle\MonologBundle\MonologBundle::class                          => ['all' => true],
-    LarpingBase\LarpingBundle\LarpingBundle::class                             => ['all' => true],
```
Comment (WilcoLouwerse) [inline api/config/bundles.php:23]: Did you already remove larpingBundle from composer.json and .lock?

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: api/config/bundles.php (hunk 1 above)
Comment (rjzondervan) [reply api/config/bundles.php:23]: Good point, will re-add this line

File: api/composer.json [hunk 2]
```diff
@@ -13,8 +13,7 @@
         "alcaeus/mongo-php-adapter": "^1.2",
         "api-platform/core": "^2.6",
         "beberlei/doctrineextensions": "^1.3",
-        "common-gateway/formio-bundle": "dev-main",
-        "commongateway/corebundle": "^1.0.89",
+        "commongateway/corebundle": "1.0.107",
```
[the remark this answers is not in the document]
Comment (WilcoLouwerse) [reply api/composer.json:16]: of cooourssee, well nevermind then 😝

[the remark this answers is not in the document]
Comment (WilcoLouwerse) [reply api/composer.json:16]: Maybe use a different symbol for this?
Keep it locked on a specific version or use a lower than 1.1.0 option ?

[the remark this answers is not in the document]
Comment (WilcoLouwerse) [reply api/composer.json:16]: <1.1.0 should work!
https://getcomposer.org/doc/articles/versions.md#summary

[the remark this answers is not in the document]
Comment (rjzondervan) [reply api/composer.json:16]: I'd lock it at the last version before 1.1.0 for now. Maybe something to look into later.

**Pull request merged**